TijanaSonjic/assignment3/wordSearch.cpp

```diff
@@ -0,0 +1,160 @@
+#include "iostream"
+#include "set"
+#include "vector"
+#include <algorithm>
+#include <string>
+#include "test_libraryr.h"
+
+using namespace std;
+
+
+
+set<string> dictionary = { "apple", "spring", "sun", "sea", "dancer" };
```

Comment:
Try to move the dictionary and the grid to be local variables of the tests, and pass them to your solution function, rather than global variables. This way the compiler would make sure that the solution does not access these globals, and would work for any dictionary/grid given to it.